packages/grafana-schema/src/common/mudball.cue

```diff
@@ -256,6 +256,8 @@ BarGaugeSizing: "auto" | "manual" @cuetsy(kind="enum")
 VizTooltipOptions: {
 	mode: TooltipDisplayMode
 	sort: SortOrder
```

Comment:
I see a significant amount of `TODO`s in some of the files we own. Are we planning on addressing this tech debt in the future?

Reply:
As far as I see, those are mostly related to documentation, they probably already added but this file left not updated 🤷🏻‍♂️

Reply:
re: doc todos with current schema system not a huge push to address those as we will eventually be replacing current schema implementation 🤷